examples/audio/main.py

```diff
@@ -49,21 +51,29 @@ def start_recording(self):
         if state == 'ready':
             self.audio.start()
 
-        if state == 'recording':
+        if state == 'recording' or state == 'resumed':
             self.audio.stop()
             self.has_record = True
 
         self.update_labels()
 
+    def pause_activity():
```

Comment:
`_activity`? Activiry is mainly used in p4a in the meaning Android Activity. This will be just confusing.